orchagent/fabricportsorch.cpp

```diff
@@ -689,6 +706,43 @@ void FabricPortsOrch::updateFabricDebugCounters()
             }
         }
 
+        // if PORT_DOWN_COUNT != PORT_DOWN_COUNT_prev:
+        //    there was a link down event in between, so we
+        //        clear skipFecErrorsOnLinkupCount
+        //        clear skipCrcErrorsOnLinkupCount
+        //        clear consecutivePollsWithErrors
+        //        clear consecutivePollsWithNoErrors
+        //        clear consecutivePollsWithFecErrs
+        //        clear consecutivePollsWithNoFecErrs
+        //
+        //        if !MANUAL_ISOLATED && AUTO_ISOLATED :
+        //              autoIsolated = 0
+        //              isolated = 0
+        //        if MANUAL_ISOLATED:
+        //              autoIsolated = 0
+        //
+        SWSS_LOG_INFO("Port %d lnk down cnt %d  handled: %d", lane, lnkDownCnt, preLnkDwnCnt);
+        if (lnkDownCnt != preLnkDwnCnt)
```

Comment:
when port goes down fabric port counters and isolation status will be cleared every polling cycle? Is this required?

Reply:
Do we need to clear the isolation status here?
should we wait for the  multiple polls without any errors after port comes up to unisolate the link?

Reply:
> rt goes down fabric port counters and isolation status will be cleared every polling cycle? Is

Not every cycle, only when handle once when the linkdownCnt != handled linkDownCnt. 
Hard to tell from local information only if this is a new card come up or not, I would do this clean up , unless the link is shutdown by user, and monitoring starts from the beginning.